Resources/Prototypes/_CD/Entities/Objects/Devices/pda.yml

```diff
@@ -0,0 +1,21 @@
+﻿- type: entity
+  parent: BasePDA
+  id: RoboticistPDA
+  name: roboticist PDA
+  description: There are exposed wires and damaged "warranty void if removed" stickers.
+  components:
+  - type: Pda
+    id: ResearchIDCard
+    state: pda-roboticist
```

Comment:
It's gonna complain about `state` not being defined on this component. Is this perhaps something that CD added to change the visuals of PDAs?